### DISCUSSION TITLE
centos7 non-Docker installed form

### DISCUSSION THREAD

**Discussion by zs001122** [milvus-io/milvus#37336] [Q&A and General discussion]
The cents7 system supports the highest version of milvus, the non-Docker installed form

**Comment by xiaofan-luan** [comment]
centos is end of time so right now rocky linux8 is recommended

**Reply by zs001122** [reply]
Thank you for your answer! Because it is required to deploy on rehat, but there are always problems with docker deployment, and I can't find specific problems, so I want to try to compile with source code.
